kasa/cli.py

```diff
@@ -656,6 +752,15 @@ async def delete_rule(dev, id):
         echo(f"No rule with id {id} was found")
 
 
+@schedule.command(name="delete_all")
```

Comment:
```suggestion
@schedule.command()
```
I think there's no need for setting the name as long as the function name can be used as a name For the `delete_rule` it was necessary to define it as the function name differs from what we want to show in the cli.

Reply:
It is also probably a good idea to add an option flag for such dangerous features. This can be done [using `prompt`](https://click.palletsprojects.com/en/8.1.x/options/#option-prompting)  for the option.

Reply:
My first time using click.